### PR TITLE
Link fixes

### PR DESCRIPTION
**Pull request by mvdan**
* Duplicate gotype
* Strip godoc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ It is intended for use with editor/IDE integration.
 
 ## Supported linters
 
-- [go vet](http://godoc.org/golang.org/x/tools/cmd/vet/) - Reports potential errors that otherwise compile.
-- [gotype](golang.org/x/tools/cmd/gotype) - Syntactic and semantic analysis similar to the Go compiler.
+- [go vet](https:///golang.org/x/tools/cmd/vet/) - Reports potential errors that otherwise compile.
```
Comment (alecthomas) [inline README.md]: Too many /'s (three rather than 2)

Comment (alecthomas) [inline README.md]: Also, this was linked to the godoc because the golang.org URLs just redirect there...seems a bit redundant to me.

Comment (mvdan) [inline README.md]: Nice catch.

True - in that case, why not use https://github.com/golang/tools/...?

File: README.md [hunk 2]
```diff
@@ -30,14 +30,13 @@ It is intended for use with editor/IDE integration.
...
 - [varcheck](https://github.com/opennota/check) - Find unused global variables and constants.
 - [structcheck](https://github.com/opennota/check) - Find unused struct fields.
-- [gotype](https://godoc.org/golang.org/x/tools/cmd/gotype) - Show compilation errors.
 - [errcheck](https://github.com/alecthomas/errcheck) - Check that error return values are used.
 - [dupl](https://github.com/mibk/dupl) - Reports potentially duplicated code.
 
 Disabled by default (enable with `--enable=<linter>`):
 
 - [testify](https://github.com/stretchr/testify) - Show location of failed testify assertions (disabled by default).
-- [test](http://golang.org/pkg/testing/) - Show location of test failures from the stdlib testing module (disabled by default).
```
Comment (alecthomas) [inline README.md]: This is the canonical link. Stripping the trailing / just redirects and adds the / back.

File: README.md (hunk 1 above)
Comment (alecthomas) [inline README.md]: Yeah I'm not sure what the canonical location for it is (ie. the github repo says "mirror"), but I guess http://golang.org/cmd/vet/ would be preferable?

PS. Thanks for catching these documentation bugs, appreciate it.

**Comment by alecthomas** [comment]
Ping. I'm happy to merge this once the minor changes have been applied.

**Comment by mvdan** [comment]
Apologies for the delay. Done.

**Comment by alecthomas** [comment]
Thanks!

**Pull request merged**